x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.tsx

```diff
@@ -16,10 +16,20 @@ import * as selectors from '../../store/hosts/selectors';
 import { useHostSelector } from './hooks';
 import { CreateStructuredSelector } from '../../types';
 import { urlFromQueryParams } from './url_from_query_params';
-import { HostMetadata, Immutable } from '../../../../../common/types';
+import { HostInfo, HostStatus, Immutable } from '../../../../../common/types';
 import { PageView } from '../components/page_view';
 import { useNavigateByRouterEventHandler } from '../hooks/use_navigate_by_router_event_handler';
 
+const HOST_STATUS_TO_HEALTH_COLOR = Object.freeze<
+  {
+    [key in HostStatus]: string;
+  }
+>({
+  [HostStatus.ERROR]: 'danger',
```

Comment:
do these terms (`danger`, `success`, `subdued`) come from the EUI component?

Reply:
@kevinlog yes - these are values that can be used for the `EuiHealth` color prop.